Code/Legacy/CryCommon/LoadScreenComponent.cpp

```diff
@@ -272,18 +268,18 @@ void LoadScreenComponent::LevelStart()
         if (pGEnv && pGEnv->pConsole)
         {
             ICVar* levelPathVar = pGEnv->pConsole->GetCVar(s_levelLoadScreenUiCanvasPath);
-            AZStd::string levelPath = levelPathVar ? levelPathVar->GetString() : "";
+            const AZStd::string levelPath = levelPathVar ? levelPathVar->GetString() : "";
 
             if (levelPath != "")
             {
                 LoadConfigSettings(s_levelFixedFpsCvarName, s_levelMaxFpsCvarName, s_levelMinimumLoadTimeCvarName);
             }
             else
             {
-                LoadConfigSettings(s_globalLevelFixedFpsCvarName, s_globalLevelMaxFpsCvarName, s_globalLevelMinimumLoadTimeCvarName);
+                // this level does not contain the load screen
+                return;
```

Comment:
If the level does not contain a loading screen, then we should load the default from s_globalLevel... 

Not sure why this was removed?

Reply:
s_globalLevelXXX cvars are described in code but never used in LY game.
Ok, I have returned them back.